Sources/AppStoreConnectCLI/Commands/TestFlight/PreReleaseVersion/ReadPreReleaseVersionCommand.swift

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import ArgumentParser
+
+struct ReadPreReleaseVersionCommand: CommonParsableCommand {
+    static var configuration = CommandConfiguration(
+        commandName: "read",
+        abstract: "Get information about a specific prerelease version.")
+
+    @OptionGroup()
+    var common: CommonOptions
+
+    @Argument(
+        help: ArgumentHelp(
+            "The app AppStore ID. eg. 432156789 or app bundle identifier. eg. com.example.App",
+            discussion: "Please input either app id or bundle Id",
+            valueName: "app-id / bundle-id"
+        ),
+        transform: Identifier.init
+    )
+    var identifier: Identifier
+
+    @Argument(
+        help: ArgumentHelp(
+            "The version no of the app",
```

Comment:
Usually we try to steer clear of shorthand especially when it comes to documentation
The apple doc description for this one is: `The version number of the prerelease version of your app.` for example